mcs/class/Microsoft.Build/Makefile

```diff
@@ -26,7 +26,7 @@ EXTRA_DISTFILES = \
 EXPR_PARSER = Microsoft.Build.Internal/ExpressionParser
 
 $(EXPR_PARSER).cs: $(EXPR_PARSER).jay $(topdir)/jay/skeleton.cs
-	(cd Microsoft.Build.Internal; $(topdir)/../jay/jay -ctv < $(topdir)/../jay/skeleton.cs ExpressionParser.jay > ExpressionParser.cs)
+	(cd Microsoft.Build.Internal; $(topdir)/../jay/jay -ctv -o ExpressionParser.cs $< < $(topdir)/../jay/skeleton.cs )
```

Comment:
Is there a shorthand for the output directory, like cd $(@D)? (that's what it is for msft nmake).